### PR TITLE
Include a few missing profile translate strings

### PR DESCRIPTION
**Pull request by ethan-nelson**
Catching a few strings on the profile page that weren't translated.

### CODE REVIEW

File: client/app/profile/profile.html [hunk 1]
```diff
@@ -60,7 +60,7 @@ <h4>{{ profileCtrl.username }}'s {{ 'Progress' | translate }}</h4>
                             </div>
                             <div>
                                 <h4>Stats</h4>
-                                <span>Time Spent Mapping (TM3 only)</span>
+                                <span>{{ 'Time Spent Mapping (TM3 only)' | translate }}</span>
```
Comment (xamanu) [inline client/app/profile/profile.html]: I'm wondering whether "TM3 only" is still something we want to print here?

Comment (xamanu) [reply client/app/profile/profile.html]: I suggest to rephrase more generally: `Time Spent Mapping (on this instance)`

**Comment by ethan-nelson** [comment]
Agreed--updated!

**Comment by ethan-nelson** [comment]
I will work on updating strings in Transifex for translation.

**Pull request merged**